components/script/dom/htmlformelement.rs

```diff
@@ -646,14 +651,28 @@ impl HTMLFormElement {
                 }
                 NodeTypeId::Element(ElementTypeId::HTMLElement(HTMLElementTypeId::HTMLOutputElement)) => {
                     // Unimplemented
-                    {}
+                    unimplemented!()
```

Comment:
Yeah, at first I thought this can panic here but later I realized it's breaking some tests. Removed it afterwards.